app/src/main/res/layout/activity_map_preference_item.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginBottom="@dimen/view_margin_common_15">
+    <LinearLayout
+        android:id="@+id/linear_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal">
+
+        <androidx.cardview.widget.CardView
+            android:id="@+id/card_view"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_margin="@dimen/view_margin_common_5"
+            app:cardCornerRadius="@dimen/map_detail_img_card_view_radius"
+            android:elevation="@dimen/map_detail_img_card_view_elevation"
+            android:layout_weight="1">
+            <ImageView
+                android:id="@+id/map_preference_card_view_image"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:contentDescription="@string/base_map_style_item_image_view_description"
+                android:layout_gravity="center"
+                android:padding="@dimen/map_detail_img_card_view_padding"
+                android:cropToPadding="true"
+                android:scaleType="centerCrop"
+                android:background="@drawable/image_border"/>
+
+        </androidx.cardview.widget.CardView>
+
+        <androidx.cardview.widget.CardView
+            android:id="@+id/card_view_2"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_margin="@dimen/view_margin_common_5"
+            app:cardCornerRadius="@dimen/map_detail_img_card_view_radius"
+            android:elevation="@dimen/map_detail_img_card_view_elevation"
+            android:layout_weight="1">
```

Comment:
If both linear layout children have weight 1, then it can be removed

Reply:
When I remove it, the columns disappear, and there is only 1 CardView per row, instead of 2.

Reply:
I see that the width is set to `match_parent` for both, see 
https://developer.android.com/develop/ui/views/layout/linear#equal-weight

Reply:
Even if it's set to 0dp there still needs to be layout_weight="1" in each one:
To create a linear layout in which each child uses the same amount of space on the screen, set the [android:layout_height] of each view to "0dp" (for a vertical layout) or the [android:layout_width] of each view to "0dp" (for a horizontal layout). Then set the [android:layout_weight] of each view to "1".